app/controllers/users_controller.rb

```diff
@@ -22,7 +22,10 @@ def update_password
     user = User.find(current_user.id)
     if params[:password] == params[:confirmation]
       user.password = params[:password]
-      if user.save
+      if params[:password] == ''
```

Comment:
Would be to cool to have a module or helper to check for a minimum length, char set, etc. for later!